js/utils.js

```diff
@@ -25,3 +38,145 @@ var convertBoolean =  function convertBoolean(text) {
   else {
   }
 };
+
```

Comment:
looks like you did some refactoring here. it would be better to do that in a separate PR since this one pertains just to CSV downloads